FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/MsalAppCredentialsTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.FunctionalTests.Configuration;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
just a comment - don't change this now! I think we were hoping to use xunit everywhere.

Reply:
@mrivera-ms  and I talked about this- we are going to delete this entire test project and migrate these tests to the functional tests repo